.github/workflows/autotools.yml

```
@@ -14,7 +29,24 @@ jobs:
+      run: make install
+    - name: list install dirs (post-install)
+      run: ls $PORTAUDIO_INSTALL_DIR $PORTAUDIO_INSTALL_DIR/include $PORTAUDIO_INSTALL_DIR/lib
+    - name: build patest_init.c (test just calls Pa_Initialize();Pa_Terminate();)
+      run: gcc -o patest_init ./test/patest_init.c -lportaudio
+    - name: run patest_init
+      run: ./patest_init
+      # test make uninstall
+    - name: uninstall
+      run: make uninstall
+    - name: list install dirs (post-uninstall)
+      run: ls $PORTAUDIO_INSTALL_DIR $PORTAUDIO_INSTALL_DIR/include $PORTAUDIO_INSTALL_DIR/lib
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:4: Double quote to prevent globbing and word splitting

(shellcheck)